ember-toucan-core/package.json

```diff
@@ -96,6 +96,7 @@
     "postcss": "^8.2.14",
     "prettier": "^2.8.3",
     "prettier-plugin-ember-template-tag": "^0.3.0",
+    "prettier-plugin-tailwindcss": "^0.2.3",
```

Comment:
Question: what's the difference between this and https://www.npmjs.com/package/ember-template-lint-plugin-tailwindcss?

Reply:
Good question! I'm not entirely sure to be honest 😅!  Poking around a little bit, my _best guess_ would be what you linked was maybe created either before the **official** `prettier-plugin-tailwindcss` was released or to be specifically used with the ember-template-lint plugin system (or maybe both?).

FWIW, it doesn't look like that package uses `prettier-plugin-tailwindcss` under the hood either.

The way our prettier rules work today is by using `overrides` and target `.hbs/.gts/.gjs` extensions rather than adding a template-lint directly it appears via https://github.com/CrowdStrike/ember-toucan-core/blob/main/ember-toucan-core/.prettierrc.cjs#L11.  This is pretty interesting!  So, as another guess, I'd imagine by leveraging `overrides` like this, we get to use the official tooling from the Tailwind folks (yay!) without having to be specific to Ember/Ember's template linting.

I apologize I don't have a better answer other than pure speculation (😂)!